integrations/Matomo/util.js

```diff
@@ -0,0 +1,316 @@
+import get from "get-value";
+import each from "@ndhoule/each";
+import logger from "../../utils/logUtil";
+import { getHashFromArray } from "../utils/commonUtils";
+
+/** If any event name matches with the goals list provided by the dashboard
+ * we will call the trackGoal with the id provided in the mapping.
+ * @param  {} event
+ * @param  {} goalListMap
+ */
+const goalIdMapping = (event, goalListMap) => {
+  each((val, key) => {
+    if (key === event.toLowerCase()) {
+      // val is the goalId provided in the dashboard
+      window._paq.push(["trackGoal", val]);
```

Comment:
We are missing the revenue parameter here

Reply:
Added